app/src/main/java/org/simple/clinic/util/RxOptional.kt

```diff
@@ -1,10 +1,11 @@
 package org.simple.clinic.util
```

Comment:
Commit: `Change filterAndUnwrapJust to ofType from filter` -> `Use ofType() in Observable#filterAndUnwrapJust()`?